File: src/main/java/cn/chahuyun/economy/model/bank/AbstractBank.java
package cn.chahuyun.economy.model.bank;

import cn.chahuyun.economy.model.bank.action.BankAction;
import lombok.Getter;
import lombok.Setter;
import net.mamoe.mirai.contact.User;

/**
 * @author Erzbir
 * @Date: 2022/11/29 22:11
 */
@Getter
@Setter
public class AbstractBank {
    private User user;

    private BankAction bankAction;

}

File: economy-core/src/main/kotlin/cn/chahuyun/economy/model/bank/AbstractBank.kt
package cn.chahuyun.economy.model.bank

import cn.chahuyun.economy.model.bank.action.BankAction
import net.mamoe.mirai.contact.User

/**
 * 银行操作上下文基类。
 *
 * user 表示当前操作的 mirai 用户，bankAction 表示本次要执行的存取/转账动作。
 * 旧版 Java API 仍会直接构造该类型，因此保持 open class 与可空字段。
 *
 * @author Erzbir
 * @Date: 2022/11/29 22:11
 */
open class AbstractBank(
    var user: User? = null,
    var bankAction: BankAction? = null
)
